### PR TITLE
Adding linux-review Gerrit instance for Prow

### PR DESCRIPTION
**Pull request by jaimehisao**
Adding configuration for the linux-review Gerrit instance, so that it can run Prow presubmit testing.

### CODE REVIEW

File: prow/oss/cluster/gerrit.yaml
Comment (chaodaiG) [inline prow/oss/cluster/gerrit.yaml:29]: is it `kunit` or `linux`?

**Review by chaodaiG — CHANGES_REQUESTED** [review]

File: prow/oss/cluster/gerrit.yaml
Comment (jaimehisao) [reply prow/oss/cluster/gerrit.yaml]: linux, my bad.

File: prow/oss/cluster/gerrit.yaml
Comment (chaodaiG) [inline prow/oss/cluster/gerrit.yaml:28]: is this still relevant? aka is this repo still under `kunit`?

Comment (bjh83) [reply prow/oss/cluster/gerrit.yaml:28]: Yep, we still use the kunit GoB instance for KUnit specific stuff. linux-review is a new Gerrit instance that we share with another team for the purpose of upstream testing (not KUnit specific stuff).

File: prow/prowjobs/linux-review.googlesource.com/linux-lkml-bridge-config.yaml
Comment (bjh83) [inline prow/prowjobs/linux-review.googlesource.com/linux-lkml-bridge-config.yaml:1]: I am pretty sure this file should be in a subdirectory: linux/kernel/git/torvalds/linux since that is the repo of the branch we are working on now.

File: prow/prowjobs/linux-review.googlesource.com/linux-lkml-bridge-config.yaml
Comment (bjh83) [inline prow/prowjobs/linux-review.googlesource.com/linux-lkml-bridge-config.yaml:2]: I think this line should be: `https://linux-review.googlesource.com/linux/kernel/git/torvalds/linux`

Since the name of this repo is `linux/kernel/git/torvalds/linux`: https://linux-review.googlesource.com/q/project:linux%252Fkernel%252Fgit%252Ftorvalds%252Flinux+status:open

**Review by bjh83 — COMMENTED** [review]
Under the `prow/prowjobs/linux-review.googlesource.com/` directory, can you put an owners file like this one here:

https://github.com/GoogleCloudPlatform/oss-test-infra/blob/master/prow/prowjobs/kunit-review.googlesource.com/linux/OWNERS

Once we become happy with lkml-gerrit bridge, we will probably want to be able to add more branches without bugging the Prow team.

**Review by chaodaiG — APPROVED** [review]
/hold
in case you want more review from your team
feel free commenting `/unhold` if you're ready

**Review by chaodaiG — APPROVED** [review]
/unhold

**Pull request merged**
